cloudformation-templates/zs_cc_cf_template_zscc_macro.yaml

```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: 2010-09-09
-Description: Zscaler Cloud Connector Pre-Deployment Macro Template
+Description: Zscaler Cloud Connector Pre-Deployment Macro Template v1.3.0
```

Comment:
If we add versioning in more than one place its just a maintenance hazard. This description is not going to appear anywhere the customer sees. The Outputs version should be the one we use